src/components/GirlsHooHack/Footer/Footer.tsx

```diff
@@ -1,8 +1,36 @@
 import React from 'react';
-
+import blueWaves from '../../../assets/images/footerWaveGHH.png';
+import whiteLogo from '../../../assets/images/whitelogo.png';
+import email from '../../../assets/images/envelope-solid.svg';
+import instagram from '../../../assets/images/instagram-square-brands.svg';
+import discord from '../../../assets/images/discord-brands.svg';
+import facebook from '../../../assets/images/facebook-f-brands.svg';
+import twitter from '../../../assets/images/twitter-brands.svg';
+import {Row} from 'react-bootstrap';
+import {isMobile} from 'react-device-detect';
+ 
 function Footer() {
   return (
-    <p>Footer</p>
+    <div>
+      <div className="bg-peacock" style={{width:"100%"}}>
+        <div className={isMobile ? "" : "pt-6 bg-white"}>
```

Comment:
if it's `isMobile ? "" : "something"`, change it to `!isMobile && "something"`

Reply:
i think without an empty string it gives an error that className can't be false